svgfilters/src/turbulence.rs

```diff
@@ -2,6 +2,9 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+use alloc::vec;
```

Comment:
This line is redundant.

Reply:
It's not. If you remove this line it wouldn't compile, because with `#[no_std]`, the std prelude is not imported automatically, and thus there is no `vec` macro in the namespace.

Reply:
Shouldn't it be marked as `#[macro_use]`?

Reply:
Marking `extern crate alloc;` with `#[macro_use]` would work, yes, but I think with 2018 edition, it's generally recommended to import them individually rather than using `#[macro_use]`.